bin/utility.py

```diff
@@ -282,10 +286,36 @@ def validateSetupSteps(self, onboard_object, wrapper_object, cli_mode='create')
                 else:
                     # valid means this waf name doesn't exists
                     logger.info(f'{onboard_object.waf_config_name:<30}{space:>20}new waf_config_name')
+                    valid_waf = False
 
         else:
             pass
 
+        # valid notify_emails is required
+        emails = onboard_object.notification_emails
+        if len(emails) == 0:
+            logger.error('At least one valid notification email is required')
```

Comment:
Will create a new bug to address this issue. Code should fail empty list of email only when activation flags are set to `true`.